File: 214/projects/10/java/FlyingBird.java
/* FlyingBird.java provides extends the bird class for flying birds
 *
 * By: Duncan Van Keulen
 * Date: 4/27/2020
 ******************************************************/

public class FlyingBird extends Bird {

    public FlyingBird() {
        super();
    }

    public FlyingBird(String name) {
        super(name);
    }

    public String movement() {
        return "flew by";
    }
}

File: 214/projects/10/ruby/FlyingBird.rb
# FlyingBird.rb | Defines the FlyingBird class which inherits attributes and methods
#   from the Bird superclass.
#
# By: Duncan Van Keulen
# Date:4/27/2020
####################################################

require './Bird.rb'

class FlyingBird < Bird

    def movement
        'flew by'
    end
end